src/applications/vaos/tests/new-appointment/components/DateTimeRequestPage.unit.spec.jsx

```diff
@@ -9,7 +9,7 @@ import { FETCH_STATUS } from '../../../utils/constants';
 import { waitFor } from '@testing-library/dom';
 import { Route } from 'react-router-dom';
 
-describe('VAOS <DateTimeRequestPage>', () => {
+xdescribe('VAOS <DateTimeRequestPage>', () => {
```

Comment:
Is this change intended?

Reply:
@Clark-W-Griswold yes, VAOS some tests were failing, so that was the recommendation from them to just skip the breaking tests

Reply:
I believe I have these fixed in master, so you could probably remove the `x` now.

Reply:
@jbalboni I removed this instance, but another was still failing, so I left that one in